database/index.js

```diff
@@ -0,0 +1,70 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    user: 'root',
+    password: '',
+    database: 'zelp'
+});
+
+connection.connect();
+
+const businessPageReviewsLoading = (businessId, callback) => {
+    let queryString = `SELECT * FROM reviews INNER JOIN users 
+                       ON reviews.UserKey = users.id
+                       WHERE reviews.Businesskey = ${businessId}`;
```

Comment:
Maybe consider using the '?'  method instead of injecting the business id straight into the script; potential security issues later on down the road  ;)